_topic_map.yml

```diff
@@ -1182,7 +1182,10 @@ Topics:
   File: architecture-component-imageregistry
 - Name: Image Registry Operator in OpenShift Container Platform
   File: configuring-registry-operator
-  Distros: openshift-enterprise,openshift-origin
+  Distros: openshift-enterprise
+- Name: Image Registry Operator in OKD
```

Comment:
Do we need the _in OpenShift Container Platform_ in line 1183 and _in OKD_ in line 1186, as you removed _on OpenShift Container Platform in line 623?

Reply:
That would be up to @bmcelvee or whoever works on that content to decide (it's in the assembly name too), maybe for a future update? I only removed it in line 623 because @Preeticp discussed changing it. For now, my main goal is just to fix the wrong product name from showing in OKD docs.